resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/kubernetes/Client.scala

```diff
@@ -64,6 +68,7 @@ private[spark] class Client(
   private val uploadedJars = sparkConf.get(KUBERNETES_DRIVER_UPLOAD_JARS).filter(_.nonEmpty)
   private val uploadedFiles = sparkConf.get(KUBERNETES_DRIVER_UPLOAD_FILES).filter(_.nonEmpty)
   uploadedFiles.foreach(validateNoDuplicateUploadFileNames)
+  checkForUserJarsFilesExistence
```

Comment:
put parentheses around zero-param methods like `checkForUserJarsFilesExistence()` -- I think this is Spark style